### PR TITLE
Rel Notes doc text for 4.7.11 release

### PR DESCRIPTION
**Pull request by jeana-redhat**
For release 19 May 2021
Preview: https://deploy-preview-32621--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-7-release-notes.html#ocp-4-7-11

### CODE REVIEW

File: release_notes/ocp-4-7-release-notes.adoc [hunk 1]
```diff
@@ -2364,3 +2364,82 @@ link:https://access.redhat.com/solutions/6001181[{product-title} 4.7.9 container
...
+
+You can now define a pre-existing AWS identity access management (IAM) role for your machine instance profiles by setting the `compute.platform.aws.iamRole` and `controlPlane.platform.aws.iamRole` fields in the `install-config.yaml` file. This allows you to match naming schemes and include predefined permissions boundaries for your IAM roles for clusters installed on AWS.
+
+[id="ocp-4-7-11-installation-pre-existing-hosted-zones-aws"]
+===== Using pre-existing Route53 hosted private zones on AWS
+
+You can now define an existing Route 53 private hosted zone for your cluster by setting the `platform.aws.hostedZone` field in the `install-config.yaml` file. You can only use a pre-existing hosted zone when also supplying your own VPC.
+
+[id="ocp-4-7-11-old-oauth-warning-message"]
+===== Informational alert for new OAuth format
```
Comment (bergerhoffer) [inline release_notes/ocp-4-7-release-notes.adoc:2408]: I would say "new OAuth token format"

File: release_notes/ocp-4-7-release-notes.adoc [hunk 2]
```diff
@@ -2364,3 +2364,82 @@ link:https://access.redhat.com/solutions/6001181[{product-title} 4.7.9 container
...
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1942027[*BZ#1942027*])
+
+* Previously, both the Cloud Credential Operator (CCO) and the Cluster Version Operator (CVO) reported if the CCO deployment was unhealthy. This resulted in double reporting if there was an issue. With this release, the CCO no longer reports if its deployment is unhealthy.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1948702[*BZ#1948702*])
+
+* Previously, namespaces were missing from the Machine Config Operator `relatedObjects` resource, and logs for some on-premise services were not collected in `must-gather` as a result. With this release, the required namespaces are added to the Machine Config Operator `relatedObjects` resource, and logs for on-premise services are collected in `must-gather`.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1950498[*BZ#1950498*])
+
+* Previously, the kubelet would sometimes overwhelm the API servers by opening a large number of watch requests for secrets and config maps, particularly on node reboot. With this release, the number of kubelet watch requests is reduced to reduce the load on API servers.
```
Comment (bergerhoffer) [inline release_notes/ocp-4-7-release-notes.adoc:2424]: Not critical to change, but "is reduced to reduce" seems repetitive, maybe there's a different term you could use for one of them?

File: release_notes/ocp-4-7-release-notes.adoc [hunk 3]
```diff
@@ -2364,3 +2364,82 @@ link:https://access.redhat.com/solutions/6001181[{product-title} 4.7.9 container
...
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1952209[*BZ#1952209*])
+
+* Previously, the cluster upstream resolver returned a DNS response that exceeded 512 bytes via UDP. As a result, CoreDNS sometimes returned the error `SERVFAIL`, logged various error messages, or both.  These errors sometimes forced the client to retry over TCP. With this release, the CoreDNS `bufsize` plugin is enabled with a UDP buffer size of 1232 bytes, and CoreDNS is less likely to return the error `SERVFAIL` or present any runtime errors when handling large DNS responses via UDP. This change also reduces the likelihood of UDP packet fragmentation.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1953097[*BZ#1953097*])
+
+* Previously, the default Google Cloud Platform (GCP) image was out of date, and referenced a version from the 4.6 release which did not support newer ignition versions. Consequently, new machines failed to boot when using the default GCP image. This update fixes the GCP image to match the release version, and new machines now properly boot with the default image.
```
Comment (bergerhoffer) [inline release_notes/ocp-4-7-release-notes.adoc:2433]: I think "Ignition" is capitalized

File: release_notes/ocp-4-7-release-notes.adoc [hunk 4]
```diff
@@ -2364,3 +2364,82 @@ link:https://access.redhat.com/solutions/6001181[{product-title} 4.7.9 container
...
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1954610[*BZ#1954610*])
+
+* Previously, when importing a devfile, the `buildguidanceimage-placeholder` container that provides the configuration for environment variables, ports, and limits was ignored. As a result, new deployments had a second container which could not start because the placeholder image could not be fetched, and the user container was missing additional configurations. This release removes the `buildguidanceimage-placeholder` container from new deployments, and adds environment variables, ports, and limit configurations to the user container so that the devfile is imported correctly.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1956313[*BZ#1956313*])
+
+* Previously, `Keepalived` did not properly clean up virtual IP (VIP) addresses when it was force restarted. As a result, VIP addresses may appear on multiple nodes, which sometimes caused problems connecting to services behind the VIP. With this release, the removal of configured VIP addresses is verified before starting `Keepalived`, and VIP addresses no longer appear on multiple nodes.
```
Comment (bergerhoffer) [inline release_notes/ocp-4-7-release-notes.adoc:2439]: may -> might

**Review by bergerhoffer — COMMENTED** [review]
Just a few comments, other than some links 404ing, which I believe is expected with these z-stream RNs.

File: release_notes/ocp-4-7-release-notes.adoc (hunk 2 above)
Comment (jeana-redhat) [reply release_notes/ocp-4-7-release-notes.adoc:2424]: I'll make one a decrease - for some reason that word escaped me earlier :)

**Pull request merged**